### PR TITLE
Java: Update MaD Declarations after Triage

### PR DESCRIPTION
**Pull request by atorralba**
This PR updates the MaD declarations after a recent triage session.

TR *TODO*

### CODE REVIEW

**Comment by kaeluka** [comment]
@atorralba do you think we can merge this so the data can be used for our evaluation? (see backref)

**Comment by atorralba** [comment]
@jcogs33 to avoid this being a self-approve, as the other person present during the triage session, do you mind giving this a rubber stamp?

File: java/ql/lib/change-notes/2023-10-31-new-models.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+category: minorAnalysis
+---
+* Added models for the following packages:
+
+  * com.google.common.io
+  * hudson
+  * hudson.console
+  * java.lang
+  * java.net
+  * java.rmi.server
```
Comment (jcogs33) [inline java/ql/lib/change-notes/2023-10-31-new-models.md:11]: There doesn't appear to be a model for this package.
```suggestion
```

**Review by jcogs33 — COMMENTED** [review]
One tiny update needed in the change note. 
Also, same question regarding whether the change note date be updated before merging?

**Comment by atorralba** [comment]
> the change note date be updated before merging?

I _think_ that date is only used to make the change note filename unique. So it shouldn't matter much.

**Review by jcogs33 — APPROVED** [review]

**Pull request merged**